Sources/CorePayments/Eligibility/EligibilityIntent.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public enum EligibilityIntent: String {
```

Comment:
Actually, I wonder if we only support `CAPTURE` and `AUTHORIZE`. Per [these PP docs](https://developer.paypal.com/docs/api/orders/v1/), these are the only 2 you can create an orderID with.

Reply:
I think it would be OK to just start with those 2. Also needs a docstring since public

Reply:
🤔 docs says `SALE` and `AUTHORIZE`, thanks for your feedback

Reply:
Updated, I also changed cases to lowercase to be more `Swifty`: [54e53c0](https://github.com/paypal/paypal-ios/pull/286/commits/54e53c0e6c6f87cc6974dd4363d2c03e4947685c)